### PR TITLE
[usage] Fix proto for CreateStripeSubscription

### PR DESCRIPTION
**Pull request by laushinka**
## Description
Adjusts the proto for CreateStripeSubscription.

## Related Issue(s)

Relates #13198

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: components/usage-api/usage/v1/billing.proto [hunk 1]
```diff
@@ -80,14 +78,10 @@ message CreateStripeCustomerResponse {
 
 message CreateStripeSubscriptionRequest {
   string attribution_id = 1;
-  string stripe_intent_id = 2;
-
-  int32 usage_limit = 3;
+  string customer_id = 2;
 }
 
 message CreateStripeSubscriptionResponse {
```
Comment (easyCZ) [inline components/usage-api/usage/v1/billing.proto:84]: Why does the subscription create not return anything?

Comment (laushinka) [reply components/usage-api/usage/v1/billing.proto:85]: As discussed, the necessary params are put back.

File: components/usage-api/usage/v1/billing.proto [hunk 2]
```diff
@@ -80,14 +78,10 @@ message CreateStripeCustomerResponse {
 
 message CreateStripeSubscriptionRequest {
   string attribution_id = 1;
-  string stripe_intent_id = 2;
-
-  int32 usage_limit = 3;
+  string customer_id = 2;
```
[the remark this answers is not in the document]
Comment (laushinka) [reply components/usage-api/usage/v1/billing.proto]: As discussed, the necessary params are put back.

File: components/usage-api/usage/v1/billing.proto [hunk 3]
```diff
@@ -80,14 +78,16 @@ message CreateStripeCustomerResponse {
...
+  int64 usage_limit = 3;
 }
 
 message CreateStripeSubscriptionResponse {
-  StripeCustomer customer = 1;
-  v1.CostCenter cost_center = 2;
+  StripeSubscription subscription = 1;
+}
+
+message StripeSubscription {
```
Comment (laushinka) [inline components/usage-api/usage/v1/billing.proto:89]: I created this subscription object that only has an ID for now. Is there a way to import the type from Stripe API itself?

Comment (easyCZ) [reply components/usage-api/usage/v1/billing.proto:89]: We don't actually want to import the Stripe ID directly, as that would couple us to Stripe on a compilation level. String is fine, it should be opaque anyway.

**Review by easyCZ — APPROVED** [review]

**Pull request merged**